.github/workflows/ci_webui.yaml

```diff
@@ -42,18 +42,22 @@ jobs:
         run: npm test
 
       - name: Lint code
+        if: ${{ !cancelled() }}
         run: npm run lint
 
       - name: Format code
+        if: ${{ !cancelled() }}
         run: |
           npm run format
           git diff --exit-code
 
       - name: Initialize CodeQL
         uses: github/codeql-action/init@v3
+        if: ${{ !cancelled() }}
```

Comment:
This should not be done. It should error out if CodeQL initialization failed. Please revert.

Reply:
That's not what this is doing. This makes it so that this steps still runs if e.g. linting fails. If this step fails, the entire job will still fail.